src/clients/firstboot_write.rb

```diff
@@ -94,10 +95,6 @@ def main
         Firstboot.default_wm
       )
 
-
-      # Dont run firstboot next time
-      SCR.Execute(path(".target.remove"), Firstboot.reconfig_file)
```

Comment:
why is this needed? what happens after next reboot?

Reply:
It is to make the workflow consistent, if you go back from the last step, and abort the installation then the write client will have already removed the reconfig_file and you will not get into the firsboot setup.

If you finish the installation then the file will be removed by the YaST2-Firstboot.service 

https://github.com/yast/yast-installation/blob/e42d5b8cde5852e913c036102e08cf8c0c18de38/package/YaST2-Firstboot.service#L15

Reply:
ok

Reply:
Other option if we do not want to rely on the Firstboot.service at all for removing it is restoring the file if going back